myapp/config/application.rb

```diff
@@ -15,5 +15,8 @@ class Application < Rails::Application
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
+    config.generators do |generator|
+      generator.orm :active_record, primary_key_type: :unsigned_integer
```

Comment:
referred [stackoverflow](https://stackoverflow.com/a/65955717) to set all table id unsigned integer